lnwallet/btcwallet/btcwallet.go

```diff
@@ -323,25 +323,21 @@ func (b *BtcWallet) Start() error {
 	err = walletdb.Update(b.db, func(tx walletdb.ReadWriteTx) error {
 		addrmgrNs := tx.ReadWriteBucket(waddrmgrNamespaceKey)
 
-		for _, keyFam := range keychain.VersionZeroKeyFamilies {
-			// If this is the multi-sig key family, then we can
-			// return early as this is the default account that's
-			// created.
-			if keyFam == keychain.KeyFamilyMultiSig {
-				continue
-			}
-
+		// Generate all accounts that we could ever need. This includes
+		// all lnd key families as well as some key families used in
+		// external liquidity tools.
+		for keyFam := uint32(1); keyFam <= 255; keyFam++ {
```

Comment:
Yes, this will be created for every wallet since there is no way to import a custom key family from the CLI.

Reply:
The how about create the accounts needed only? Not sure about adding this data to every lnd node without the node operator's awareness. And this only adds accounts under 1017/1 right?

Reply:
The downside to only adding those we actually need instead of just the first 255 ones is that we keep a list of Loop/Pool accounts/key families hard coded in `lnd`. And if we ever add a new key family to a product we first need to add it to `lnd`. With just using 255 accounts/families we just need to keep any new family below that number.
I think it's a reasonable tradeoff in exchange for an additional ~300kB of data.

> And this only adds accounts under 1017/1 right?

Yes, this only affects the `1017'` key scope.

Reply:
Only downside I can think of here is that the initial call to list accounts will list all these other accounts, but that can be addressed primarily with documentation, or if we filter out the ones w/o any active addresses.